public/assets/harvard.css

```diff
@@ -1124,10 +1124,10 @@ form.form-inline #from_year0, form.form-inline #to_year0 {
   }
   .modal-button {
     display: inline-block;
-    margin-left: 75%;
+    /* margin-left: 75%; */
```

Comment:
Why is this commented rather than removed?  If there's a reason it's left in, please add that in a comment above, otherwise, I think it's better practice to just remove the line - it's all in git, after all, if we need to see what it used to be we can look back in history.